src/SimpleGraphs/simpledigraph.jl

```diff
@@ -245,9 +245,7 @@ is_directed(g::SimpleDiGraph) = true
 is_directed(::Type{SimpleDiGraph}) = true
 is_directed(::Type{SimpleDiGraph{T}}) where T = true
 
-
-function has_edge(g::SimpleDiGraph{T}, e::SimpleDiGraphEdge{T}) where T
-    s, d = T.(Tuple(e))
+function has_edge(g::SimpleDiGraph{T}, s, d) where T
     verts = vertices(g)
     (s in verts && d in verts) || return false  # edge out of bounds
     @inbounds list = g.fadjlist[s]
```

Comment:
why is this `@inbounds`? I don't think it matters much. Do benchmarks say otherwise?

(I see we're doing it below, but we should avoid annotations if they don't do anything.)